components/admin-panel/sections/ActivityLog/ActivityTypeFilter.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+
+import { ActivityClasses } from '../../../../lib/constants/activities';
+import { ActivityCategoryLabelI18n } from '../../../../lib/i18n/activities';
+
+import { StyledSelectFilter } from '../../../StyledSelectFilter';
+
+const ActivityTypeFilter = ({ onChange, value, ...props }) => {
+  const intl = useIntl();
+  const getOption = value => ({ label: intl.formatMessage(ActivityCategoryLabelI18n[value]), value });
+
+  const ActivityTypeKeys = Object.keys(ActivityClasses);
+  ActivityTypeKeys.unshift('ALL');
+
+  return (
+    <StyledSelectFilter
+      inputId="activity-type-filter"
+      onChange={({ value }) => onChange(value)}
+      value={getOption(value || 'ALL')}
+      options={ActivityTypeKeys.map(getOption)}
```

Comment:
As mentioned by PM, we're not interested in all activity classes: let's filter out reports

Reply:
Thanks, I've filtered out reports. 👍🏽